CODEOWNERS

```diff
@@ -12,29 +12,29 @@
 # These are the owners (approvers) for localization contents
 # in each `/content/language/` directory.
 
-# Approvers for Korean contents
-/content/ko/ @seokho-son @Eviekim @jihoon-seo @yunkon-kim
-
-# Approvers for Portuguese contents
-/content/pt-br/ @edsoncelio @brunoguidone @jessicalins
+# Approvers for Arabic contents
+/content/ar/ @TarekMSayed @same7ammar @AShabana @hacktron95
 
-# Approvers for Hindi contents
-/content/hi/ @Garima-Negi @sayantani11 @anubha-v-ardhan @jayesh-srivastava
+# Approvers for Bengali contents
+/content/bn/ @mitul3737 @Mouly22 @ikramulkayes
 
 # Approvers for German contents
 /content/de/ @CathPag @iamNoah1 @DaveVentura
 
+# Approvers for Spanish contents
+/content/es/ @CathPag @raelga @electrocucaracha
+
+# Approvers for Hindi contents
+/content/hi/ @Garima-Negi @sayantani11 @anubha-v-ardhan @jayesh-srivastava
+
 # Approvers for Italian contents
 /content/it/ @Giulia-dipietro @meryem-ldn @annalisag-spark @sistella
```

Comment:
@iamNoah1 

It is because the invitations for @AShabana and @Giulia-dipietro are pending.
If they don't accept it, we cannot give write permissions to them. I think we need to check whether they are still willing to  be an approver.

Reply:
Thanks ! :)